src/tests/test_utils.rs

```diff
@@ -14,3 +18,38 @@ pub fn validate_execute_invalid_id_field(
         },
     }
 }
+
+pub fn setup_asset_marker(
+    base_address: String,
+    access_address: String,
+    marker_denom: String,
+) -> QueryMarkerResponse {
+    let expected_marker: MarkerAccount = MarkerAccount {
+        base_account: Some(BaseAccount {
+            address: base_address,
+            pub_key: None,
+            account_number: 10,
+            sequence: 0,
+        }),
+        manager: "".to_string(),
+        access_control: vec![AccessGrant {
+            address: access_address,
+            permissions: vec![1, 2, 3, 4, 5, 6, 7],
+        }],
+        status: MarkerStatus::Active.into(),
+        denom: marker_denom,
+        supply: "1000".to_string(),
+        marker_type: MarkerType::Restricted.into(),
```

Comment:
maybe this should be an argument, if we need to define the `Coin` ones as well

maybe make a separate function for ease for the restricted marker.
```
pub fn setup_restricted_asset_marker(
    base_address: String,
    access_address: String,
    marker_denom: String,
) -> QueryMarkerResponse {
   setup_asset_marker(base_address, access_address, marker_denom, MarkerType::Restricted)
}
```